### PR TITLE
Add action types as constants in one single file

### PR DESCRIPTION
**Pull request by zelol**
I opted for the 1) solution. I prefer simplicity :)

Let me know if everything is okay.

### CODE REVIEW

File: src/actions.js [hunk 1]
```diff
@@ -1,64 +1,69 @@
 import { fetchCocktailEnrichment } from "./services/cocktailDBAPI.service";
 
+import { LOAD_COCKTAILS, LOAD_INGREDIENTS, LOAD_GLASSES, UPDATE_FILTER,
```
Comment (mikeyhogarth) [inline src/actions.js]: Would it be okay to include these as;
```
import * as actionTypes from './actionTypes';
```
and then refer to them like;
```
return { type: actionTypes.LOAD_COCKTAILS, payload };
```
Same in the other file - this big import at the top isn't going to scale and it means we'll need to change code in two places every time a new action type comes along.

File: src/reducers/index.js [hunk 2]
```diff
@@ -2,6 +2,11 @@ import produce from "immer";
 import { loadPersistedState } from "../utilities/persistence";
 import { hasDialog } from "../filterConfig";
 
+import { LOAD_COCKTAILS, LOAD_INGREDIENTS, LOAD_GLASSES, UPDATE_FILTER,
```
Comment (mikeyhogarth) [inline src/reducers/index.js]: As per other comment;
```
import * as actionTypes from '../actionTypes'
```

**Comment by mikeyhogarth** [comment]
Cheers dude - couple of comments.

File: src/actions.js (hunk 1 above)
Comment (zelol) [reply src/actions.js]: You're right! I'll work on it tomorrow. Thanks for the feedback.

**Comment by zelol** [comment]
Done.

**Comment by mikeyhogarth** [comment]
Nice one thanks dude 😀

**Comment by zelol** [comment]
You're welcome!
Thank you for giving me the opportunity to do my first PR on someone's project. Your app is cool, it makes me want to drink even if I never drink haha.
If you have others "good first issues" let me know please :)

**Comment by mikeyhogarth** [comment]
Will do guy, may this PR be the first of many for you!

**Pull request merged**